tests/tests/fv3_gsd_dtend_debug

```diff
@@ -99,7 +105,5 @@ export LSM=3
 export LSOIL_LSM=9
 export KICE=9
 
-export PRINT_DIFF_PGR=.true.
```

Comment:
No need to change this now, but since we remove it from this test we should add it to some other test before our code changes go back to the authoritative repositories. For example the `FV3_GSD_noah` test. The challenge will be to remember doing it.

Reply:
You could open an issue?

Reply:
Done: https://github.com/NOAA-GSL/ufs-weather-model/issues/71